### ISSUE TITLE
Cache warming form CLI does not queue with --queue=1

### ISSUE THREAD

**Issue by pelmered** [putyourlightson/craft-blitz#294]
### Describe the bug
Queue option does not work from CLI for `blitz/cache/warm` and `blitz/cache/refresh`

The following two commands runs synchronously and does not add the cache warming to the queue:
```bash
php craft blitz/cache/refresh --queue=1
php craft blitz/cache/warm --queue=1
```

### To reproduce
Steps to reproduce the behaviour:
1. Run `php craft blitz/cache/warm --queue=1` from command line. 

### Expected behaviour
The cache warming should be added to the queue and CLI prompt should be returned almost immediately like it does when you click the Blitz Action buttons in Craft admin. 

### Screenshot
This should be queued and I should not see the progress bar:
![Screenshot 2021-02-25 at 15 44 59](https://user-images.githubusercontent.com/680058/109170263-e8315b00-7780-11eb-9a89-b358f127af6f.png)

### Versions
- Plugin version: 3.7.0
- Craft version: 3.6.7

**Comment by bencroker** [comment]
Thanks, fixed in https://github.com/putyourlightson/craft-blitz/commit/7eb888ac286117fbbda1b8d34c4c89fbaabd6d27 for the next release.

**Comment by pelmered** [comment]
Awesome @bencroker! Thanks for the very quick fix!

**Comment by bencroker** [comment]
Released in 3.8.0.
